packaging/installer/install-required-packages.sh

```diff
@@ -721,6 +721,20 @@ declare -A pkg_gcc=(
   ['default']="gcc"
 )
 
+# g++, required for building protobuf
+# All three cases of this not being required are systems that implicitly
+# include g++ when installing gcc.
+declare -A pkg_gxx=(
```

Comment:
should we add protobuf as well?

Reply:
Not if we are planning on bundling them.

I do need to add them as dependencies for the DEB/RPM packages though.